main.py

```diff
@@ -17,8 +17,8 @@
 
 
 # config_path should be overriden when running `bliss` poetry executable
-# e.g., `bliss --cp case_studies/summer_template -cn config`
-@hydra.main(config_path="bliss/conf", config_name="base_config", version_base=None)
+# e.g., `bliss --cp case_studies/summer_template`
+@hydra.main(config_path="case_studies/summer_template", config_name="config", version_base=None)
```

Comment:
This probably shouldn't default to `case_studies/summer_template` since that's just a template config